crates/aranya-client-capi/src/api/defs.rs

```diff
@@ -928,3 +992,197 @@ pub unsafe fn afc_recv_data(
 
     Ok(true)
 }
+
+/// Query devices on team.
+///
+/// @param client the Aranya Client [`Client`].
+/// @param team the team's ID [`TeamId`].
+/// @param devices returns a list of device IDs on the team [`DeviceId`].
+/// @param devices_len returns the length of the devices list [`DeviceId`].
+///
+/// @relates AranyaClient.
+pub fn query_devices_on_team(
+    client: &mut Client,
+    team: &TeamId,
+    devices: Option<&mut MaybeUninit<DeviceId>>,
+    devices_len: &mut usize,
+) -> Result<(), imp::Error> {
+    let client = client.deref_mut();
+    let data = client
+        .rt
+        .block_on(client.inner.queries(team.0).devices_on_team())?;
+    let data = data.__data();
+    if let Some(devices) = devices {
+        let out = aranya_capi_core::try_as_mut_slice!(devices, *devices_len);
+        for (dst, src) in out.iter_mut().zip(data) {
+            dst.write(DeviceId(*src));
+        }
+        *devices_len = data.len();
+        return Ok(());
+    }
+    *devices_len = data.len();
+    Ok(())
+}
+
+/// The formula for computing the amount of characters in a base64 string from the original number of bytes is:
+/// base64_str_len = (bytes*1375)/1000
+///
+/// A DeviceId is 64 bytes.
+///  
+/// ARANYA_DEVICE_ID_STR_LEN is the number of characters required to hold the base64 string plus the null terminator:
+/// (sizeof(AranyaDeviceId)*1375)/1000+1 = 89
+// TODO: compute size from size_of::<DeviceId>()
```

Comment:
We don't need all of this context in the public API docs. How about just:

```rust
/// The size in bytes of a [`DeviceId`] converted to a human-readable string.
// <your explanation here>
```

NB: `///` vs `//`